model/src/main/java/com/epam/esm/audit/AuditEntity.java

```diff
@@ -0,0 +1,91 @@
+package com.epam.esm.audit;
```

Comment:
Why do you need it?

Reply:
I need this to save a record in db for AuditListener operations such as creation, updation, deletion of an entity